keras_nlp/models/f_net/f_net_backbone.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""FNet backbone model."""
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.layers.fnet_encoder import FNetEncoder
```

Comment:
We have `fnet_encoder` and `f_net_backbone`. I think we should pick one style and go with it.

Reply:
f_net is the literal following of our style guide. I added a good first issue to fix the existing path.